server-data/resources/[framework]/rsg-inventory/config.lua

```diff
@@ -9,4 +10,385 @@ Config.MaxInventorySlots = 41 -- Max inventory slots for a player
 Config.CleanupDropTime = 15 * 60 -- How many seconds it takes for drops to be untouched before being deleted
 Config.MaxDropViewDistance = 12.5 -- The distance in GTA Units that a drop can be seen
 Config.UseItemDrop = false -- This will enable item object to spawn on drops instead of markers
-Config.ItemDropObject = `prop_nigel_bag_pickup` -- if Config.UseItemDrop is true, this will be the prop that spawns for the item
\ No newline at end of file
+Config.ItemDropObject = `sf_prop_sf_backpack_01a` -- if Config.UseItemDrop is true, this will be the prop that spawns for the item
+
+Config.Progressbar = {
+    Enable = false,         -- True to Enable the progressbar while opening inventory
+    minT = 350,             -- Min Time for Inventory to open
+    maxT = 500              -- Max Time for Inventory to open
+}
+
+Config.VendingObjects = {
+    "prop_vend_soda_01",
+    "prop_vend_soda_02",
+    "prop_vend_water_01"
+}
+
+-- TODO: Implement Bin Objects
```

Comment:
So the bin objects don't have any relevance in the script. Should I just remove it fullheartedly?